.github/workflows/release-please.yaml

```diff
@@ -0,0 +1,18 @@
+name: Release Please
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: googleapis/release-please-action@v4
+        with:
+          release-type: terraform-module
```

Comment:
I'll be interested interested to see what this will update after it merges
![image](https://github.com/user-attachments/assets/6bf19dc8-46d6-4191-b18a-c5b72aa36e87)